### ISSUE TITLE
Add digests to relevant files

### ISSUE THREAD

**Issue by chrahunt** [pypa/get-pip#47]
From pypa/pip#5658, we could provide digests so users can verify that get-pip.py was downloaded correctly.

One use case could be during manual configuration of some server that does not have root CA certificates installed:

1. Execute `curl --insecure -o get-pip.py https://bootstrap.pypa.io/get-pip.py` on the server
2. Run `sha256sum get-pip.py`
3. Navigate to `https://bootstrap.pypa.io/get-pip.py.sha256` in a web browser
4. Compare the contents of that page with the output of `sha256sum`

**Comment by pradyunsg** [comment]
Same as #41.

**Comment by chrahunt** [comment]
Not necessarily. That will result in a `*.asc` next to the file containing the signature, which serves a different use case. Compare the digests and signatures on the [Python download page](https://www.python.org/downloads/release/python-374/), for example.

**Comment by pradyunsg** [comment]
Indeed. I misread in a hurry. Sorry!
